shared/common-adapters/section-list.stories.tsx

```diff
@@ -27,16 +27,16 @@ const customRowRender = ({item}) => (
   </Text>
 )
 
-const small = []
+const small: Array<unknown> = []
```

Comment:
what's `unknown`? Why are you using it here?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type
in terms of when i'm fixing things up for real vs punting usually involves a hierarchy like this (most important short term to least)

1. Reducers
2. Actions
3. Connected components
4. Stories